Garcia-Julz/mine-your-business
# Jg adding 1

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]
## Type of change

Please delete options that are not relevant.
# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-adding-1`
4. go to user 1 or 3 and add a ticket. you should see your newly created ticket after you. submit the form.
# Checklist:
